xbmc/windows/GUIWindowSystemInfo.cpp

```diff
@@ -161,6 +161,10 @@ void CGUIWindowSystemInfo::FrameMove()
     }
 
     SetControlLabel(i++, "{} {}", 22010, SYSTEM_GPU_TEMPERATURE);
+
+#if defined(TARGET_ANDROID)
```

Comment:
I would prefer if this feature would need no platform "ifdeffery", backing what thexai said.